src/rules/form-control-has-label.ts

```diff
@@ -42,8 +49,10 @@ const rule: Rule.RuleModule = {
   create(context) {
```

Comment:
hey, can you add the `scheme` about the option you just added? I think we got an example in `label-has-for`, I don't know if it's super necessary, but I think the rules that have option have those

Reply:
Assuming you mean the `schema`? No worries, updated.